api.d.ts

```diff
@@ -12,6 +12,7 @@ export interface PullRequest {
   pendingReviews: string[];
```

Comment:
This data type was written when we didn't exactly have good mocks. I don't think this maps to what we need anymore. eg. we need to know when the review was approved. The UI now shows outgoing/incoming very differently and we should probably represent them differently.